src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3092,7 +3092,7 @@ public static void DCS_MyPersonSurrogate()
     [Fact]
     public static void DCS_FileStreamSurrogate()
     {
-        const string TestFileName = "Test.txt";
+        string TestFileName = Path.GetTempFileName();
```

Comment:
Nit: Also, now that `TestFileName` is a local rather than a const, it should be renamed to `testFileName`.